faker/providers/address/ko_KR/__init__.py

```diff
@@ -4,14 +4,171 @@
 
 
 class Provider(AddressProvider):
-    city_suffixes = ('마을', '시', '군', '구')
-    building_number_formats = ('#', '####', '###')
-    street_suffixes = ('동', '로', '거리')
-    postcode_formats = ('###-###', '###-###')
-    states = (
-        '서울특별시', '부산광역시', '대구광역시', '인천광역시', '광주광역시', '대전광역시', '울산광역시',
-        '세종특별자치시', '경기도', '강원도', '충청북도', '충청남도', '전라북도', '전라남도', '경상북도',
-        '경상남도', '제주특별자치도')
+    """Korea has two address and postal code system.
+
+    ## Address
```

Comment:
Instead of markdown, can you use ReST Formatting for headers, lists and links? see http://docutils.sourceforge.net/docs/user/rst/quickstart.html for the syntax